peer_name_hash.go

```diff
@@ -43,10 +44,12 @@ func PeerNameFromBin(nameByte []byte) PeerName {
 	return PeerName(hex.EncodeToString(nameByte))
 }
 
-// Bin encodes PeerName as a byte slice.
-func (name PeerName) Bin() []byte {
+// bytes encodes PeerName as a byte slice.
+func (name PeerName) bytes() []byte {
 	res, err := hex.DecodeString(string(name))
-	checkFatal(err)
+	if err != nil {
+		panic(fmt.Sprintf("unable to decode name to bytes: %q", string(name)))
```

Comment:
`%q` does add quotes and escaping, which is super-handy in more complex cases (e.g. " " vs "\t"), but for this case it's probably overkill - changed to concatenation.